cpp/ql/lib/semmle/code/cpp/internal/ResolveGlobalVariable.qll

```diff
@@ -24,8 +24,8 @@ private predicate isGlobalWithMangledNameAndWithoutDefinition(@mangledname name,
  * a unique global variable `complete` with the same name that does have a definition.
  */
 private predicate hasTwinWithDefinition(@globalvariable incomplete, @globalvariable complete) {
-  not variable_instantiation(incomplete, complete) and
   exists(@mangledname name |
+    not variable_instantiation(incomplete, complete) and
     isGlobalWithMangledNameAndWithoutDefinition(name, incomplete) and
```

Comment:
Should be kept.

Reply:
Fixed in fe2c806c2db5f3dceaf2877f34a25fcbce34337b